### PR TITLE
CHAOS-1960: adds chaos linux fault tooltips

### PR DESCRIPTION
**Pull request by StrawberryPad**

### CODE REVIEW

File: src/Tooltip/TooltipDictionary.yaml [hunk 1]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    content: Number of worker processes to start. Corresponds to the number of CPU cores to consume.
+    width: "400"
+chaos_tune_fault_duration:
+    content: Duration through which chaos is injected into the target resource (in seconds).
+    width: "400"
+chaos_tune_fault_ramptime:
+    content: Period to wait before and after injecting chaos (in seconds).
+    width: "400"
+chaos_tune_fault_processids
+    content: Comma-separated Ids of the target processes.
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6834]: Comma-separated target process Ids.

To keep it consistent with the next tooltip.

File: src/Tooltip/TooltipDictionary.yaml [hunk 2]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    content: Whether to force kill the process using the SIGKILL signal or SIGTERM signal for graceful killing.
+    width: "400"
+chaos_tune_fault_services
+    content: Names of the target services.
+    width: "400"
+chaos_tune_fault_selfhealingservices
+    content: Set to true if the service restarts on its own upon stopping it.
+    width: "400"
+chaos_tune_fault_interval
+    content: Duration of a single iteration of chaos (in seconds). Should be less than or equal to the input for "Duration".
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6852]: **Duration** as it is a UI element

File: src/Tooltip/TooltipDictionary.yaml [hunk 3]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    content: Time offset to increment or decrement the system time. Should be provided in (+/-)[numeric-hours]h[numeric-minutes]m[numeric-seconds]s format.
+    width: "400"
+chaos_tune_fault_disablentp
+    content: Set it to disable the NTP server.
+    width: "400"
+chaos_tune_fault_memory
+    content: Amount of memory to be consumed.
+    width: "400"
+chaos_tune_fault_networklatency
+    content: Amount of latency to add to connection in ms. For example: 2000
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6864]: is ms milliseconds?

File: src/Tooltip/TooltipDictionary.yaml [hunk 4]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    width: "400"
+chaos_tune_fault_networkinterface
+    content: The network interface to target. For example: eth0
+    width: "400"
+chaos_tune_fault_destinationhosts
+    content: List of the target hostnames or keywords. For example: ["google.com","litmuschaos.io"]
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6870]: For example,

Comment (roshnisarangadharan) [reply src/Tooltip/TooltipDictionary.yaml:6870]: global

File: src/Tooltip/TooltipDictionary.yaml [hunk 5]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    width: "400"
+chaos_tune_fault_destinationips
+    content: List of the target IPs. For example: ["1.1.1.1","8.8.8.8"]
+    width: "400"
+chaos_tune_fault_packetlosspercentage
+    content: Percentage of packet loss. For example: 100
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6876]: For example, 100.

File: src/Tooltip/TooltipDictionary.yaml [hunk 6]
```diff
@@ -6818,6 +6818,105 @@ chaos_lci_log_file_max_age:
...
+    width: "400"
+chaos_tune_fault_packetcorruptionpercentage
+    content: Percentage of packets to corrupt. For example: 50
```
Comment (roshnisarangadharan) [inline src/Tooltip/TooltipDictionary.yaml:6879]: For example, 50.

**Review by roshnisarangadharan — APPROVED** [review]

**Comment by StrawberryPad** [comment]
Closing. This is replaced by PR [#349](https://github.com/harness/ng-tooltip/pull/349)

**Comment by StrawberryPad** [comment]
Closing. This is replaced by PR https://github.com/harness/ng-tooltip/pull/349

**Pull request closed without merging**